transport-udt/src/main/java/io/netty/channel/udt/nio/NioUdtByteConnectorChannel.java

```diff
@@ -186,4 +190,33 @@ public InetSocketAddress localAddress() {
     public InetSocketAddress remoteAddress() {
         return (InetSocketAddress) super.remoteAddress();
     }
+
+    private static void privilegedBind(final SocketChannelUDT socketChannel, final SocketAddress localAddress)
+            throws IOException {
+        try {
+            AccessController.doPrivileged(new PrivilegedExceptionAction<Void>() {
+                @Override
+                public Void run() throws IOException {
+                    socketChannel.bind(localAddress);
+                    return null;
+                }
+            });
+        } catch (PrivilegedActionException e) {
+            throw (IOException) e.getCause();
+        }
+    }
+
+    private static boolean privilegedConnect(final SocketChannelUDT socketChannel, final SocketAddress remoteAddress)
+            throws Exception {
+        try {
+            return AccessController.doPrivileged(new PrivilegedExceptionAction<Boolean>() {
+                @Override
+                public Boolean run() throws IOException {
+                    return socketChannel.connect(remoteAddress);
+                }
+            });
+        } catch (PrivilegedActionException e) {
+            throw (IOException) e.getCause();
+        }
```

Comment:
It's not ready for it, but I've introduced it in #6224 which is a first step towards getting ready for it. 😇